solr/core/src/java/org/apache/solr/handler/component/QueryComponent.java

```diff
@@ -401,6 +403,14 @@ public void process(ResponseBuilder rb) throws IOException
     doProcessUngroupedSearch(rb, cmd, result);
   }
 
+  private int getMinExactHits(SolrParams params) {
+    long minExactHits = params.getLong(CommonParams.MIN_EXACT_HITS, Integer.MAX_VALUE);
```

Comment:
I plan to move the default value to a solrconfig config in a future PR, for now, MAX_VALUE (disabled)